src/api_server/swagger/firecracker.yaml

```diff
@@ -577,6 +652,9 @@ definitions:
         description: Flag for enabling/disabling Hyperthreading
       cpu_template:
         $ref: "#/definitions/CpuTemplate"
+      track_dirty_pages:
+        type: boolean
+        description: Enable dirty page tracking. If this is enabled, then incremental VM snapshots can be created.
```

Comment:
I think the description can be improved by differentiating between full/diff snapshot - sparse file.

Reply:
Done.